src/main/java/com/wise23/chariteed/model/Patient.java

```diff
@@ -1,4 +1,4 @@
-package com.wise23.chariteed;
+package com.wise23.chariteed.model;
 
 public class Patient {
```

Comment:
If this is a part of database, then
@Entity

Reply:
If i add `@Entity` I get the error message: 

`Error creating bean with name 'entityManagerFactory' defined in class path resource [...] No identifier specified for entity: com.wise23.chariteed.model.Patient`

Do you know anything about this?

Reply:
@JannesVolkens do u already have a database for this application?

This happens, because it requires a link to actual table in a database.
In particular, it needs further annotations like 
@Table(name="patients")
and fileds require smth like
@Column(name="email")